frontend/alert/components/AlertForm/index.tsx

```diff
@@ -101,14 +101,19 @@ const AlertForm = ({
     setTimeline,
     autofillSection = "",
 }: AlertFormProps) => {
-    const [section, setSection] = useState(autofillSection);
+    const [selectedCourses, setSelectedCourses] = useState<Set<Section>>(
+        new Set()
```

Comment:
Yeah I thought of that but `Set` is type `Section` while` autofillSection` is `string` so I thought it would be easier to just add a special case. Is there anything we can do instead that would work be better?

Reply:
Hmm yeah ig that's fine then. Maybe it would be worth doing an API request for the info of the default section, but that would prob just add extra lag. The special case logic isn't too substantial anyway.

Reply:
Sounds good!